package indigo.pb;

option java_package = "ru.indigosystem.serv.pb";

option java_outer_classname = "IndigoPB";
option optimize_for = LITE_RUNTIME;

message indigo_msg {
        optional indigo_UUID device_id = 1;

        optional int32 id = 2 [ default = 0 ];

        extensions 100 to max;
}

message internal_msg {
        optional int32 id = 1 [ default = 0 ];

        extensions 100 to max;
}


// A 128-bit id, transferred as two 64-bit longs
message indigo_UUID {
    optional sint64 least_significant_bits = 1 [ default = 0 ];
    optional sint64 most_significant_bits = 2 [ default = 0 ];
}

